tests/unit/builders/address.ts

```diff
@@ -0,0 +1,18 @@
+export function randomAddress(): string {
+  const characters = '0123456789abcdef';
+  let address = '0x';
+
+  for (let i = 0; i < 40; i++) {
+    const randomIndex = Math.floor(Math.random() * characters.length);
+    address += characters[randomIndex];
+  }
+
+  return address;
+}
+
+// popular token addresses used in unit tests
+export const zeroAddress = '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE';
```

Comment:
This isn't the 'ZeroAddress' this is the address for the native asset on Ethereum and some other chains. I'd rename it the EtherAddress or NativeAssetAddress.

Reply:
Thanks for the clarification, renamed.